packages/contracts-bedrock/scripts/fpac/README.md

```diff
@@ -0,0 +1,41 @@
+# `fpac-deploy`
+
+Chain-ops scripts for the Fault Proof Alpha Chad contracts.
+
+## Dependencies
+
+* [just](github.com/casey/just)
```

Comment:
Do we really need to introduce another build tool for this? Seems like we could just use make?

Much as I'm not a fan of make, I think having even more different build systems and dependencies is even worse.

Reply:
Agree with this, in a separate repository like kuna or axos, I think it's acceptable to use a different build tool as long as it's consistent. We should also be consistent in the monorepo and use make.